src/version.h

```diff
@@ -28,7 +28,7 @@
   #define VERSION "undefined"
 #endif
 
-void version(std::ostream& out = std::cout) {
+void printVersions(std::ostream& out = std::cout) {
   out << VERSION << std::endl;
```

Comment:
Let's start the versions text with "zim-tools"

Reply:
https://github.com/kiwix/kiwix-tools/pull/516#discussion_r777203400 is an answer to this comment too

Reply:
Like on kiwix-tools, finally changed my mind.